app/jinja_filters.py

```diff
@@ -80,7 +82,9 @@ def format_unit(
     return formatted_unit
 
 
-def format_unit_input_label(unit: str, unit_length: str = "short") -> str:
+def format_unit_input_label(
+    unit: str, unit_length: Literal["short", "long", "narrow"] = "short"
```

Comment:
the literal is repeated in a few places, TypeAlias?

Reply:
+1 to this

Reply:
Updated 👍